EventFilter/SiPixelRawToDigi/src/PixelDataFormatter.cc

```diff
@@ -193,22 +213,53 @@ void PixelDataFormatter::interpretRawData(bool& errorsInEvent, int fedId, const
     }
     if unlikely(skipROC) continue;
     
-    
-    int dcol = (ww >> DCOL_shift) & DCOL_mask;
-    int pxid = (ww >> PXID_shift) & PXID_mask;
+
     int adc  = (ww >> ADC_shift) & ADC_mask;
-    
-    LocalPixel::DcolPxid local = { dcol, pxid };
-    if unlikely(!local.valid()) {
-      LogDebug("PixelDataFormatter::interpretRawData") 
-	<< "status #3";
-      errorsInEvent = true;
-      errorcheck.conversionError(fedId, &converter, 3, ww, errors);
-      continue;
-    }
-    
-    GlobalPixel global = rocp->toGlobal( LocalPixel(local) );
+    LocalPixel *local=NULL;
```

Comment:
Isn't `local` leaking memory? I don't see it de-allocated. Why not use std::unique_ptr instead of a bare pointer?